src/views/List.jsx

```diff
@@ -6,12 +6,11 @@ export function List({ data }) {
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
+
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{data.map((data) => (
+					<ListItem name={data.name} />
```

Comment:
When mapping over an array, you need to give each array item a key 😊
There's an error in the console that warns you about that.
<img width="595" alt="image" src="https://user-images.githubusercontent.com/17270662/230048948-09d425af-7c77-47ea-aef0-3ce04596e9fc.png">

You can read more about it [here](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key).